routers/routers.go

```diff
@@ -6,7 +6,7 @@ import (
 	"net/http"
 )
 
-func SetRoutes() {
+func SetRoutes() { 
```

Comment:
I'm not a big fan of `SetRoutes()` setting global state. In main.go you call `http.ListenAndServe(port, nil)`, and it's not easy for me to figure out where the handler is coming from. I'd prefer something like
```go
// In routers.go
func GetRoutes() http.Handler {
  mux := http.NewServeMux()
  mux.HandleFunc(paths.LoginOrSignupPath, handlers.HandleLoginOrSignupRequest)
  mux.HandleFunc(paths.LoginOrSignupPath, handlers.HandleLoginOrSignupRequest)
  mux.HandleFunc("/user", handlers.HandleUserRequest)
  mux.HandleFunc("/session", handlers.HandleSessionRequest)
  return mux
}

// In main.go
func main() {
  handler := routers.GetRoutes()
  http.ListenAndServe(port, handler)
}
```

GetRoutes might be a terrible name, but you get the idea